examples/service-clients/azure-client/external-controller/src/presence.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IPresence,
+	Latest,
+	LatestMap,
+	type PresenceStates,
+	type PresenceStatesSchema,
+} from "@fluid-experimental/presence";
+
+import type { DieValue } from "./controller.js";
+
+export interface DiceValues {
+	die1?: DieValue;
+	die2?: DieValue;
+}
+
+const statesSchema = {
+	// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+	lastRoll: Latest({} as DiceValues),
+	// Both lastRoll and lastDiceRolls are not required. lastDiceRolls is here
```

Comment:
Maybe drawing some points on what each one (lastRoll, lastDiceRolls) offers, advantages, constraints etc. It's nice to use examples and get to "aha" moments :) especially for someone coming from persisted-state use-case scenarios.